app/controllers/auctions_controller.rb

```diff
@@ -7,7 +7,7 @@ def index
 
     respond_to do |format|
       format.html do
-        sam_status_message_for(flash)
+        @auctions.sam_status_message_for(flash)
```

Comment:
I will admit this confused me at first because I expected from Rails conventions that `@auctions` would be an array or something and not an instance of `AuctionsIndexViewModel`. I think it's fine for this PR, but do you think we should rename to `@auction_collection`? Does it cause any confusion in the relevant view to do so?